### PR TITLE
Add downloads page

### PR DESCRIPTION
**Pull request by wasnotrice**
This is my first date with Webmachine, so tell me, is this a good way to handle our offsite downloads? We provide a shoesrb.com URL, then redirect to an external file.

### CODE REVIEW

**Comment by wpp** [comment]
:+1: sweet! After this is merged I'll try to make it a bit prettier.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # IDEs
 /.idea
 
+.rvmrc
```
Comment (steveklabnik) [inline .gitignore:4]: I generally think this is a bad idea, but I don't use rvm anymore, so I guess I don't care. :wink:

**Comment by steveklabnik** [comment]
Seems good! :heart:

@wpp pretty away!

File: .gitignore (hunk 1 above)
Comment (PragTob) [inline .gitignore:4]: Hm I also like to have the .rvmrc right in the repository... :o

Comment (wasnotrice) [inline .gitignore:4]: Feel free to revert. My intent was to make it easier for someone to work with whatever ruby they have on their system. For this project, it shouldn't really matter.

Certainly this line doesn't belong in this commit ;)

**Pull request merged**